src/views/LayoutWithRouting.tsx

```diff
@@ -33,13 +33,18 @@ const SIDENAVBAR_ITEMS: NavItemType[] = [
   },
 ]
 
+const StudioView = loadable(() => import('@/studio/StudioView'))
```

Comment:
What happens when the view takes some time to load? We won't see this now but if the bundle size grows, it can take non-instant amount of time to fetch the bundle. We probably need some kind of fallback for when that happens

Reply:
@drillprop I see you've added a fallback both into `loadable` and `Suspense`. Do we need both though? Which of those will get displayed? Or will they both get screen time?

Reply:
Fixed, I did it by mistake. I was experimenting with both and forgot to remove Suspense. `fallback` from loadable should be enough.

Reply:
:+1: